openapi/integration/aws/donations/get-posts.json

```diff
@@ -12,7 +12,8 @@
       "responseParameters": {
         "method.response.header.Access-Control-Allow-Methods": "'GET'",
         "method.response.header.Access-Control-Allow-Headers": "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token'",
-        "method.response.header.Access-Control-Allow-Origin": "'*'"
+        "method.response.header.Access-Control-Allow-Origin": "'*'",
+        "method.response.header.Cache-Control": "'max-age=300'"
```

Comment:
shouldn't it be rendered from terraform

Reply:
We have to do some work there. ticket: https://github.com/CraftsmenLtd/BloodConnect/issues/318